### DISCUSSION TITLE
How to migrate the existing database to ChistaDATA Cloud?

### DISCUSSION THREAD

**Discussion by emoido** [ChistaDATA/Cloud#6] [❓FAQ] [answered]
How to migrate the existing database to ChistaDATA Cloud?

**Comment by TheraniAA** [comment, marked as answer]
While there are tools available for online migration, the steps involved in migrating from an existing database to ChistaDATA Cloud can vary depending on the source database type. To ensure a smooth migration process, it is advisable to consult the ChistaDATA Cloud documentation or seek assistance from the ChistaDATA support team.
